sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update.hpp

```diff
@@ -302,12 +304,12 @@ bool test_acc(queue q, const Config &cfg) {
         for (int cnt = 0; cnt < cfg.repeat; ++cnt) {
           if constexpr (n_args == 0) {
             if constexpr (UseMask) {
-              if constexpr (UseProperties)
+              if constexpr (UseProperties || DataSizeRequiresLSC)
```

Comment:
this never gets called if !UseProperties && DataSizeRequiresLSC, we do a constexpr return in this case earlier but the compiler still tries to instantiate these for some reason, so just make it happy